crates/rspack_core/src/compiler/mod.rs

```diff
@@ -73,7 +73,7 @@ impl Compiler {
     options: CompilerOptions,
     plugins: Vec<BoxPlugin>,
     buildtime_plugins: Vec<BoxPlugin>,
-    output_filesystem: Option<Box<dyn WritableFileSystem + Send + Sync>>,
+    output_filesystem: Option<Box<dyn WritableFileSystem>>,
```

Comment:
These two traits are used in a multi-thread runtime. I think they can not be removed.
I have no more problem if this is modified. Do you need a double check? @h-a-n-a

Reply:
https://github.com/web-infra-dev/rspack/blob/6b0e319a620873a14c6ce319a2009ba4297b8aef/crates/rspack_fs/src/write.rs#L8-L42